package.json

```diff
@@ -107,19 +108,19 @@
     "react-dom": "15.5.4",
     "react-ga": "1.2.0",
     "react-hot-loader": "1.2.8",
+    "react-number-format": "2.0.4",
     "react-onclickout": "2.0.6",
     "react-router": "3.0.5",
     "react-tappable": "0.8.0",
     "rootpath": "0.1.2",
     "sass-loader": "3.2.0",
     "scroll-behavior": "0.4.0",
-    "sinon": "1.17.5",
     "style-loader": "0.13.0",
     "underscore": "1.8.3",
     "validator": "7.2.0",
-    "webpack": "1.15.0",
+    "webpack": "2.7.0",
     "webpack-bundle-analyzer": "2.9.0",
-    "webpack-dev-server": "1.10.1",
+    "webpack-dev-server": "2.7.1",
```

Comment:
Couldn't we port this to devDependencies and change the way it's imported in the server.js file so when EB runs npm install --production it doesn't need to install devDeps?

Reply:
True, will update that. Ran into a problem deploying to AWS because of that reason, previously.